### DISCUSSION TITLE
Json Schema support custom type with custom validation

### DISCUSSION THREAD

**Discussion by MuraraAllan** [ory/kratos#1193] [Q&A] [answered]
Do identity schema support custom type and custom validation for types ?

**Comment by zepatrik** [comment, marked as answer]
What do you mean by custom type? You can specify any identity schema using JSON Schema draft 7 https://json-schema.org/specification.html

**Comment by vinckr** [comment]
Hey @MuraraAllan,

Just checking in if your question was answered or if there was still something unclear.
Marking this as answered for now.

Thanks a ton! 🐝

**Comment by MuraraAllan** [comment]
Thank you so much, sorry for the delay in response.

Thank you so much @vinckr @zepatrik
